include/aspect/initial_composition/random_perturbation.h

```diff
@@ -0,0 +1,96 @@
+/*
+  Copyright (C) 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_initial_composition_random_perturbation_h
+#define _aspect_initial_composition_random_perturbation_h
+
+#include <aspect/initial_composition/interface.h>
+#include <aspect/simulator_access.h>
+
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
```

Comment:
As far as I know there is no need for `DEAL_II_DISABLE_EXTRA_DIAGNOSTICS` anymore and also `DEAL_II_ENABLE_EXTRA_DIAGNOSTICS`. (keep the include).